x-pack/plugins/security/server/plugin.ts

```diff
@@ -267,8 +295,19 @@ export class Plugin {
 
         return this.authenticationStart;
       },
+      getAnonymousAccessService: () => {
+        if (!this.anonymousAccessStart) {
+          throw new Error('AnonymousAccess service is not started!');
+        }
+
+        return this.anonymousAccessStart;
+      },
     });
 
+    core.capabilities.registerProvider(() => ({
```

Comment:
__note:__ I was hesitating about this one since it has nothing to do with the feature controls (FC) and UI capabilities are mainly used with FC right now. But if I understand the idea behind Core's/OSS UI capabilities then it's not necessarily related to the FC or Security in general, so I thought that anonymous access can be treated as a capability that may affect the way we render UI. 

Am I just looking for excuse or you think it's a valid use case for UI capabilities as well?